src/hooks/useLock.ts

```diff
@@ -9,10 +9,17 @@ const hiiqContractConfig = {
   abi: hiIQABI,
 }
 
-const erc20ContractConfig = {
-  address: config.iqAddress as `0x${string}`,
-  abi: erc20Abi,
-}
+//use WIQ for dev
+const erc20ContractConfig =
+  config.alchemyChain === 'iqChain'
+    ? {
+        address: config.wiqAddress as `0x${string}`,
```

Comment:
if you are gonna be doing this if/else condition across all site, just do a config.erc20IQ in the config and use that. Then you will only have that condition there and code will be more clean